frontend/src/pages/Home/components/JoinUsForm/Form.jsx

```diff
@@ -1,8 +1,40 @@
 import React from "react";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import * as yup from "yup";
```

Comment:
please use `joi-browser` instead of `yup` in order to be consistent everywhere.

Reply:
Hey! @Jassi10000 as we have already used `joi` for validation in other forms. So , I request you to implement the same for this form as well. Also we are using `joi` in backend as well so it will be more consistent to go with `joi`

Reply:
So now I should delete all the changes with react-hook-form and replace them with Joi
Sure, will try to do that as soon as possible